### PR TITLE
add llava gpu example

### PR DESCRIPTION
**Pull request by Ricky-Ting**
### 1. Why the change?

In this PR, we add GPU example of LLaVA using bigdl-llm's optimize_model API.

https://github.com/analytics-zoo/nano/issues/632#issuecomment-1786618228

### CODE REVIEW

File: python/llm/example/GPU/PyTorch-Models/Model/llava/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# LLaVA
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on LLaVA models on [Intel GPUs](../README.md). For illustration purposes, we utilize the [liuhaotian/llava-v1.5-13b](https://huggingface.co/liuhaotian/llava-v1.5-13b) as a reference LLaVA model.
```
Comment (shane-huang) [inline python/llm/example/GPU/PyTorch-Models/Model/llava/README.md:2]: maybe change to llava-7b as the reference model.

File: python/llm/example/GPU/PyTorch-Models/Model/llava/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+- `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: argument defining the huggingface repo id for the LLaVA model (e.g. `liuhaotian/llava-v1.5-13b` to be downloaded, or the path to the huggingface checkpoint folder. It is default to be `'liuhaotian/llava-v1.5-13b'`.
+- `--image-path-or-url IMAGE_PATH_OR_URL`: argument defining the input image that the chat will focus on. It is required.
+- `--n-predict N_PREDICT`: argument defining the max number of tokens to predict. It is default to be `512`.
+
+If you encounter some network error (which means your machine is unable to access huggingface.co) when running this example, refer to [Trouble Shooting](#4-trouble-shooting) section.
+
+
+#### Sample Output
+#### [liuhaotian/llava-v1.5-13b](https://huggingface.co/liuhaotian/llava-v1.5-13b)
```
Comment (shane-huang) [inline python/llm/example/GPU/PyTorch-Models/Model/llava/README.md:55]: use 7b as a reference example.

File: python/llm/example/CPU/PyTorch-Models/Model/llava/generate.py [hunk 3]
```diff
@@ -58,7 +58,7 @@
 
 from bigdl.llm import optimize_model
 
-# Lod the pretained model.
+# Lod the pretrained model.
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/PyTorch-Models/Model/llava/generate.py:61]: Lod -> Load

File: python/llm/example/GPU/PyTorch-Models/Model/llava/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# LLaVA
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on LLaVA models on [Intel GPUs](../README.md). For illustration purposes, we utilize the [liuhaotian/llava-v1.5-7b](https://huggingface.co/liuhaotian/llava-v1.5-7b) as a reference LLaVA model.
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/PyTorch-Models/Model/llava/README.md:2]: As we have also verified llama 13b on Arc, we may use the format [here](https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/llama2) which includes both 7 and 13b models but use the 7b one as the default reference model.

Comment (Oscilloscope98) [reply python/llm/example/GPU/PyTorch-Models/Model/llava/README.md:2]: This may include minor changes in description, argument info, sample output, etc.

**Review by shane-huang — COMMENTED** [review]
LGTM.

**Review by shane-huang — APPROVED** [review]

**Pull request merged**